### PR TITLE
Mark a few tests as parallel

### PR DESCRIPTION
**Pull request by bmbouter**
[noissue]

### CODE REVIEW

**Review by fao89 — APPROVED** [review]

File: pulpcore/tests/functional/api/using_plugin/test_crud_repos.py [hunk 1]
```diff
@@ -360,6 +361,7 @@ def test_headers(self):
         self.remotes_api.partial_update(self.remote.pulp_href, data)
 
 
+@pytest.mark.parallel
 class CreatePulpLabelsRemoteTestCase(unittest.TestCase):
```
Comment (mdellweg) [inline pulpcore/tests/functional/api/using_plugin/test_crud_repos.py:365]: Maybe it's a question for another day, but do they need to inherit from `unittest.TestCase`?

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
Maybe we should introduce the ci changes via the plugin_template (that is when they are merged). Or they will be overruled by the next automatic update.

File: pulpcore/tests/functional/api/using_plugin/test_crud_repos.py (hunk 1 above)
Comment (bmbouter) [reply pulpcore/tests/functional/api/using_plugin/test_crud_repos.py:365]: They don't need to, but I plan to handle that when I come through and actually port the tests to pytest in another PR.

**Comment by bmbouter** [comment]
Actually I can declare these marks from pulp-smash! https://github.com/pulp/pulp-smash/pull/1268

**Pull request merged**